src/materialized/src/version_check.rs

```diff
@@ -115,8 +123,11 @@ struct V1VersionRequest<'a> {
     status: Status,
 }
 
+/// General status of the materialized server, for telemetry
 #[derive(Serialize)]
 struct Status {
+    /// Unique token for every time materialized is restarted
+    session_id: Uuid,
```

Comment:
Doesn't need to block this PR, but this doesn't seem like the right place to document the API of the telemetry server. That probably deserves an actual specification that lives in a markdown file or somesuch.

Reply:
Fair, yes.